src/main/service/sjpFilterService.ts

```diff
@@ -104,23 +121,48 @@ export class SjpFilterService {
         return activeFilters;
     }
 
-    private doFiltering(allCases, postcodeFilters, prosecutorFilters) {
+    private doFiltering(allCases, londonPostalAreaCodes, postcodeFilters, prosecutorFilters) {
```

Comment:
'londonPostalAreaCodes' does not need to be passed into this method. It is a class constant so you can just used it.